test/streamly-tests.cabal

```diff
@@ -364,28 +364,29 @@ test-suite FileSystem.Handle
     buildable: False
 
 test-suite Network.Inet.TCP
-  import: test-options
+  import: lib-options
   type: exitcode-stdio-1.0
   main-is: Streamly/Test/Network/Inet/TCP.hs
-  if flag(use-streamly-core)
-    buildable: False
-  -- XXX on Windows these test cases are hanging for ever
-  -- need to be investigated.
-  -- https://github.com/composewell/streamly/issues/2315
   if os(windows)
+    ghc-options:-rtsopts -fno-ignore-asserts
+  else
+    ghc-options:-rtsopts -fno-ignore-asserts -threaded -with-rtsopts=-N
+  include-dirs: .
```

Comment:
Why is this required now, and was not required earlier?

Reply:
It came from test-options.